### PR TITLE
English grammar fixes, punctuations, etc

### PR DESCRIPTION
**Pull request by Smooey**
Fixes for following bug reported threads:

https://community.woltlab.com/thread/247350-ready-grammar-use-punctuation-wcf-general-areas/

https://community.woltlab.com/thread/247707-ready-grammar-use-punctuation-acp-maintenance-rebuild-data/

https://community.woltlab.com/thread/247348-language-wcf-acp-group-option-category-mod/

https://community.woltlab.com/thread/247090-language-wcf-acp-option-user-cleanup-notification-lifetime/

https://community.woltlab.com/thread/247614-language-wcf-acp-paidsubscription-usergroups-wcf-acp-paidsubscription-usergroups/

### CODE REVIEW

File: wcfsetup/install/lang/en.xml [hunk 1]
```diff
@@ -802,13 +802,13 @@ Examples for medium ID detection:
 		<item name="wcf.acp.option.session_enable_virtualization.description"><![CDATA[Users can log-in from multiple devices (PC, tablet, smartphone, etc) concurrently, all devices will share the same session. Disabling this option will restrict users to be logged-in with only one device at once, causing every login to terminate sessions used by the user’s other devices. It is not recommended to disable this option and you should be well aware of the consequences if you decide to disable it.]]></item>
 		<item name="wcf.acp.option.show_clock"><![CDATA[Show current time in page footer]]></item>
 		<item name="wcf.acp.option.timezone"><![CDATA[Timezone]]></item>
-		<item name="wcf.acp.option.timezone.description"><![CDATA[Default timezone of your page]]></item>
-		<item name="wcf.acp.option.user_online_timeout"><![CDATA[User-Online timeout]]></item>
+		<item name="wcf.acp.option.timezone.description"><![CDATA[The default timezone of your page.]]></item>
+		<item name="wcf.acp.option.user_online_timeout"><![CDATA["Users Online" Timeout]]></item>
```
Comment (TimWolla) [inline wcfsetup/install/lang/en.xml]: Please use the proper unicode quotes: 

http://www.fileformat.info/info/unicode/char/201c/index.htm
http://www.fileformat.info/info/unicode/char/201d/index.htm

**Comment by TimWolla** [comment]
Apart from the quotation marks it looks good to me. Thanks! I'll let @dtdesign have another look.

**Comment by Smooey** [comment]
Thanks Tim, the quotation marks were / are used else where too for "Users Online", hence adding them there. :) 

eg: options > forum > forum list > general > Display “Users Online” list
options > module > users> “Users Online” list
etc.

**Comment by Gravatronics** [comment]
@Smooey You used different marks than at the positions you mentioned:

yours: `"Users Online"`
the others: `“Users Online”`

**Comment by Smooey** [comment]
Oh! Doh, thanks for pointing it out.. lmao. I see now, I missed Tim's other comment. Fix those in local computer, resync to my github, then pull request again?

**Comment by Gravatronics** [comment]
@Smooey No, just change the quotation marks, commit the changes and push/sync the changes to your fork of the repository on GitHub. This pull request will automatically be updated.

**Comment by Smooey** [comment]
I think I got it now, I'm new to github so you'll have to bare with me here.. :P

**Pull request merged**